cpp/tests/stream_compaction/stream_compaction_tests.cu

```diff
@@ -93,7 +93,7 @@ void BooleanMaskTest(cudf::test::column_wrapper<T> source,
                      cudf::test::column_wrapper<cudf::bool8> mask,
                      cudf::test::column_wrapper<T> expected)
 {
-  gdf_column result;
+  gdf_column result{};
   EXPECT_NO_THROW(result = cudf::apply_boolean_mask(source, mask));
 
   EXPECT_TRUE(expected == result);
```

Comment:
I think this line showcases where this PR could go further. Shouldn't `column_wrapper<T>::operator==()` also compare names? Then we would also make sure that `expected` contains a nullptr for name.

Reply:
Updated the `gdf_equal_columns` method in cudf_test_utils.cu